src/Controller/EventController.php

```diff
@@ -53,15 +54,22 @@ public function index()
     public function add()
     {
         $errors = [];
+        $userData = [];
 
         if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+
+            $userData = $_POST;
+            $textFilter = new Text();
+            $textFilter->setTexts($userData);
+            $userData = $textFilter->filter();
+
             $eventManager = new EventManager($this->getPdo());
             $event = new Event();
 
-            if (!$this->formErrors()) {
+            if (!$this->formErrors($userData)) {
```

Comment:
le ! (not) est l'inversion sous entendu false mais ton formErrors retourne un array... A priori tu ne rentreras jms dans cette condition car un array est toujours retourné (meme vide).